README.md

```diff
@@ -36,15 +36,59 @@ LLM App reads a corpus of documents stored in S3 or locally, preprocesses them,
+| `local`         | This mode runs the application using Huggingface Transformers, which eliminates the need for external APIs. It provides a convenient way to use state-of-the-art NLP models locally without additional services.                                                                   |
 
 ### Installation
 
+
 - **Clone the repository:** This is done with the `git clone` command followed by the URL of the repository:
     ```bash
     git clone https://github.com/pathwaycom/llm-app.git
     ```
 
+- **Environment Variables:** Create an .env file in `llm_app/` directory and add the following environment variables, adjusting their values according to your specific requirements and setup.
+
+    | Environment Variable        | Description                                                                                                                                                                                                                                                |
+    | --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | PIPELINE_MODE               | Determines which pipeline to run in your application. Available modes are [`contextful`, `contextful_s3`, `contextless`, `local`]. By default, the mode is set to `contextful`.                                                                            |
+    | PATHWAY_REST_CONNECTOR_HOST | Specifies the host IP for the REST connector in Pathway. For the dockerized version, set it to `0.0.0.0` Natively, you can use `127.0.0.1`. interfaces.                                                                                                    |
+    | PATHWAY_REST_CONNECTOR_PORT | Specifies the port number on which the REST connector service of the Pathway should listen. Here, it is set to 8080.                                                                                                                                       |
+    | OPENAI_API_TOKEN            | The API token for accessing OpenAI services. If you are not running the local version, please remember to replace it with your personal API token, which you can generate from your account on [openai.com](https://platform.openai.com/account/api-keys). |
+    | PATHWAY_CACHE_DIR           | Specifies the directory where cache is stored. You could use /tmp/cache.                                                                                                                                                                                   |
+    
+
+    ```bash
+    PIPELINE_MODE=contextful
```

Comment:
I think I'd prefer mode as an argument to the script (as earlier). IMHO, env vars are good for settings that do not alter the logic much and it applies to all other values but the mode changes the behavior completely.
However, you can disagree.

Reply:
Let's do it in a separate pr